rewrite-java/src/main/java/org/openrewrite/java/cleanup/SimplifyDurationCreationUnits.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2022 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.cleanup;
+
+import org.openrewrite.Applicability;
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Recipe;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.internal.lang.Nullable;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.JavaTemplate;
+import org.openrewrite.java.MethodMatcher;
+import org.openrewrite.java.search.UsesMethod;
+import org.openrewrite.java.tree.Expression;
+import org.openrewrite.java.tree.J;
+import org.openrewrite.java.tree.JavaType;
+
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import java.time.temporal.TemporalUnit;
+
+public class SimplifyDurationCreationUnits extends Recipe {
+
+    private enum DurationUnits {
+        // order is important; factors are tried in this order (largest-to-smallest)
+        // (the Java spec states that `.values()` uses declaration order)
+        DAYS("ofDays", ChronoUnit.DAYS),
+        HOURS("ofHours", ChronoUnit.HOURS),
+        MINUTES("ofMinutes", ChronoUnit.MINUTES),
+        SECONDS("ofSeconds", ChronoUnit.SECONDS),
+        MILLIS("ofMillis", ChronoUnit.MILLIS);
+
+        final String methodName;
+        final long millisFactor;
+        final MethodMatcher methodMatcher;
+
+        DurationUnits(String methodName, TemporalUnit unit) {
+            this.methodName = methodName;
+            this.millisFactor = Duration.of(1, unit).toMillis();
+            this.methodMatcher = new MethodMatcher("java.time.Duration " + methodName + "(long)");
+        }
+    }
+
+    @Override
+    public String getDisplayName() {
+        return "Simplify java.time.Duration units";
```

Comment:
```suggestion
        return "Simplify `java.time.Duration` units";
```
Can use markdown in recipe titles and descriptions